Yafc.Model/Serialization/PropertySerializers.cs

```diff
@@ -300,8 +311,8 @@ internal class DictionarySerializer<TOwner, TCollection, TKey, TValue>(PropertyI
     public override void SerializeToJson(TOwner owner, Utf8JsonWriter writer) {
         TCollection? dictionary = getter(owner);
         writer.WriteStartObject();
-        foreach (var (key, value) in dictionary.Select(x => (Key: KeySerializer.GetJsonProperty(x.Key), x.Value))
-            .OrderBy(x => x.Key, StringComparer.Ordinal)) {
+
+        foreach (var (key, value) in dictionary.Select(x => (Key: KeySerializer.GetJsonProperty(x.Key), x.Value)).OrderBy(x => x.Key, StringComparer.Ordinal)) {
```

Comment:
Sure. If you prefer it with new lines after those, then it can be made so.

Reply:
Hmmm, now that I think about it, there was quite a number of cases where those three went together on one line. I think because they fit thematically. Please let me know if this argument does not resolve this point.